test/fake_http_server.h

```diff
@@ -68,9 +68,13 @@ class FakeServer {
   // responses over a hanging GET.
   void AllowStream(const std::string& path);
 
-  // Blocks until at least one client has connected to the given path.
-  // Returns false if the timeout is reached with no client connections.
-  bool WaitForOneStreamWatcher(const std::string& path, time::seconds timeout);
+  // Blocks until the total number of connections to the given path
```

Comment:
Done. (Left the comment though.)

Reply:
I cannot recognize whether `(including connections that have since been closed)` is for `min_connections` or for `the total number of connections`. Suggest moving it to earlier part.
```
Blocks until the total number of connections (including connections that have since been closed) to the given path has reached at least min_connections.
```

Reply:
Done. (Thanks!)